src/app/pages/home/home.component.html

```diff
@@ -14,6 +14,7 @@ <h1>Micronutrient Action Policy Support (MAPS) Tool</h1>
       <h2>MAPS will be a web-hosted tool, communicating estimates of dietary micronutrient (MN) supplies and deficiency
         risks at national and sub-national scales in Africa. </h2>
       <button mat-raised-button color="primary" [routerLink]="ROUTES.MAPS_TOOL | route">Maps Tool Portal</button>
+      <br><br><img src="../../../assets/images/maps-logo-full-greyscale.svg" alt="BMGF MAPS Logo" class="logo">
```

Comment:
/assets better?

Reply:
Having a margin would be better than a couple of <br> I think

Reply:
1. Yup :)
2. Indeed